server/schemas/schemaController.js

```diff
@@ -67,12 +67,29 @@ module.exports = {
    */
   createSchema: function(request, response) {
```

Comment:
Looks good conceptually. Since `createSchema` would do more than forward to the `s3Cache` method with same name, the method name and documentation needs to be updated to reflect the differences.